pkg/policy/api/rule.go

```diff
@@ -590,3 +633,31 @@ func (kr *PortRuleKafka) GetAPIVersion() (int16, bool) {
 
 	return *kr.apiVersionInt, false
 }
+
+// MapRoleToAPIKey maps the Role to the low level set of APIKeys for that role
+func (kr *PortRuleKafka) MapRoleToAPIKey() bool {
+	if len(kr.Role) <= 0 {
+		return false
+	}
+
+	if len(kr.apiKeyInt) > 0 {
```

Comment:
What is this check for? Shouldn't `len(kr.apiKeyInt)` always be  0 at this point?

Reply:
You are right. It will be 0 always.

Reply:
done